src/styles/globals.css

```
@@ -1155,3 +1155,725 @@
+  font-family: var(--font-inter), -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
+}
+
+/* Header */
+.mdm-head {
+  padding: 18px 24px 14px;
+  border-bottom: 1px solid var(--color-border-subtle);
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  background:
+    linear-gradient(180deg, rgba(167, 139, 250, 0.04), transparent 80%),
```

Stylelint (17.11.0)

[error] Expected "0.04" to be "4%" (alpha-value-notation)

(alpha-value-notation)